scripts/kernel_build_conf.json

```diff
@@ -0,0 +1,19 @@
+{
+  "email_from": "dkaushik@emerson.com",
+  "email_to": "dkaushik@emerson.com",
+  "kernel_build": {
+    "kernel_src_dir": "nilrt-kernel-build/linux",
+    "target_host": "10.152.8.54",
+    "target_user": "admin",
+    "arch": "x86_64",
+    "temp_modules_dir": "nilrt-kernel-build/linux/tmp-glibc/modules",
+    "repo_url": "https://github.com/ni/linux.git",
+    "target_branch": "nilrt/master/6.12",
+    "stable_rt_remote": "https://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-stable-rt.git",
+    "nilrt_root": "nilrt-sdk",
+    "toolchain_prefix": "/usr/local/oecore-x86_64/sysroots/x86_64-nilrtsdk-linux/usr/bin/x86_64-nilrt-linux/x86_64-nilrt-linux-",
+    "required_packages": ["bc", "bison", "flex", "gcc", "git", "kmod", "libelf-dev", "libssl-dev", "make", "u-boot-tools"],
```

Comment:
This is unlikely to change frequently, right? If so, you can just add the list of packages in the code instead of the config.